packages/core/drizzle/0000_youthful_sumo.sql

```diff
@@ -0,0 +1,172 @@
+CREATE SCHEMA "latitude";
+--> statement-breakpoint
+CREATE TABLE IF NOT EXISTS "latitude"."accounts" (
```

Comment:
what's this table for? oauth? I see some mentions to provider

Reply:
This is for OAuth. It is recommended to still add it even if we're not using it, for compatibility with NextAuth providers and easier OAuth integration in the future.